Modules/documents/Controller/DocumentslistController.php

```diff
@@ -63,28 +63,26 @@ public function indexAction($id_space) {
     public function editAction($id_space, $id) {
         $this->checkAuthorizationMenuSpace("documents", $id_space, $_SESSION["id_user"]);
         $lang = $this->getLanguage();
-
         $model = new Document();
         $data = $model->get($id_space, $id);
 
         $form = new Form($this->request, "DocumentsEditAction");
         $form->setTitle(DocumentsTranslator::Edit_Document($lang));
         $form->addText("title", DocumentsTranslator::Title($lang), true, $data["title"]);
         $form->addUpload("file_url", DocumentsTranslator::File($lang));
-        $form->setValidationButton(CoreTranslator::Save($lang), "documentsedit/" . $id_space);
+        $form->setValidationButton(CoreTranslator::Save($lang), "documentsedit/" . $id_space . "/" . $id);
         $form->setCancelButton(CoreTranslator::Cancel($lang), "documents/" . $id_space);
 
         if ($form->check()) {
             $title = $this->request->getParameter("title");
             $id_user = $_SESSION["id_user"];
             $idNew = $model->set($id, $id_space, $title, $id_user);
```

Comment:
don't you create a new entry in db here?

Reply:
(in  if ($form->check()) block:
if $id > 0 => update existing entry entry
else => create new entry

Reply:
so idnew will be the new id or the existing id, no ? at least this is what set method should do

Reply:
and in this case, no need of iddoc

Reply:
ah yes, I created $idDoc before passing $id in the request parameters... But I can remove it